content/en/blog/general/introducing-kyverno-envoy-plugin/index.md

```diff
@@ -0,0 +1,232 @@
+---
+date: 2024-06-04
+title: Kyverno-Envoy-Plugin - Kyverno policies based authorization plugin for Envoy 
+linkTitle: Kyverno-Envoy-Plugin - Kyverno policies based authorizaiton plugin for Envoy
+author: Sanskar Gurdasani
+description: Make external authorization easy with kyverno-envoy-plugin.
+---
+
+![Kyverno-Envoy-Plugin](logo.png)
+
+While microservices offer benefits by breaking applications into smaller, independent parts, they also introduce new challenges offcourse microservies improve productivity of individual development. Managing service discovery, authentication, and authorization across multiple microservices can be complex. These tasks become even more intricate due to the diverse and short-lived nature of microservice environments.
+
+As more organizations move to using microservices, there is an increasing need for separate authentication and authorization mechanisms that work across different microservices.
+
+In this blog post, we will introduce [kyverno-envoy-plugin](https://github.com/kyverno/kyverno-envoy-plugin), how it works and you can use this version of kyverno to enforce fine-grained, context-aware access control policies with Envoy without modifying your microservice or application code.
```

Comment:
Words like "Kyverno", "Envoy", "Kubernetes" and others are proper nouns in English. They must be titled cased. Please make sure this is consistent throughout the post.